calc/policy_sorter.go

```diff
@@ -112,6 +113,20 @@ func (p PolKV) String() string {
 	return fmt.Sprintf("%s(%s)", p.Key.Name, orderStr)
 }
 
+func (p *PolKV) GovernsApplyOnForward() bool {
+	if p.Value != nil {
```

Comment:
Minor: think the golang convention is to handle the nils first with an early return:
```#!go
if p.Value == nil {
    return false
}
if p.Value.ApplyOnForward == nil {
    // If ApplyOnForward flag pointer is nil, this policy has been created by old version of calico.
    // Apply on forward traffic as default.
    return true
}
return *p.Value.ApplyOnForward
```